geometry_msgs/Point centroid
float32 diameter
---
lr_msgs/Pose2D[] goal_poses
string[] goal_types
bool success
string message
